### ISSUE TITLE
Add Pylint to Travis Build

### ISSUE THREAD

**Issue by jonathan-j-lee** [BerkeleyAutomation/malasakit-v1#36]
Currently, `pylint` fails badly with the code (e.g. -12.60/10 for `views.py`). However, once the code base is cleaned up, we will want to enforce good style.

**Comment by JustinMi** [comment]
Also we should all install `pylint` so we can slowly fix all the style errors

**Comment by jonathan-j-lee** [comment]
Update: to avoid false positives when standard `pylint` inspects Django-specific code, we should use the `pylint-django` plugin: 
1. Install with `pip2 install pylint-django`.
2. Use with `pylint --load-plugins pylint_django path/to/file.py`.

**Comment by jonathan-j-lee** [comment]
This should now be addressed in commit a305f5dd2f049ad023b9d159b3e55b44bb0c73f0. Once this branch is merged into master, I will close this issue.

Once a source file is cleaned up, please add it the `Makefile` as `LINT_TARGETS=pcari/models.py pcari/tests.py [...]` (that is, a space-separated list of source files, assuming the working directory is `malasakit-django`).

**Comment by jonathan-j-lee** [comment]
Addressed in #50.
